renderer/utils/sagas.js

```diff
@@ -571,6 +576,7 @@ export default function* rootSaga() {
     takeEvery('UPLOAD_CODE', uploadStudentCode),
     takeEvery('TOGGLE_FIELD_CONTROL', handleFieldControl),
     takeEvery('TIMESTAMP_CHECK', timestampBounceback),
+    takeEvery('EXPORT_RUN_MODE', exportRunMode),
```

Comment:
Do you know if there is already a `EXPORT_RUN_MODE ` action? If not, down the line, I think we need to add one so that there is a dispatch for the saga to take this action.

Reply:
`EXPORT_RUN_MODE` is just meant to be the channel for sending the `studentCodeStatus` (e.g. `TELEOP`, `ESTOP`) from Ansible to Runtime. Currently there is no dispatch because I was still considering whether it needs to be sent continuously (similar to `ansibleGamepads`), or just when requested (similar to the now retired `TIMESTAMP_CHECK`).